### PR TITLE
vxstream ignore error unsupported file type

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/26767

## Description
Added the `dontThrowErrorOnFileDetonation` argument in **crowdstrike-submit-url*

## Screenshots
image

## Does it break backward compatibility?
   - [ ] Yes
       - Further details: 
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CrowdStrikeFalconSandbox/Playbooks/playbook-Detonate_URL_-_CrowdStrike.yml [hunk 1]
```diff
@@ -274,21 +349,34 @@ inputs:
...
     simple: "30"
   required: false
   description: How much time to wait before a timeout occurs  (minutes)
+  playbookInputQuery:
+- key: dontThrowErrorOnFileDetonation
+  value:
+    simple: "false"
+  required: false
+  description: |-
+    Should the playbook should fail on unsupported file type?
```
Comment (Itay4) [inline Packs/CrowdStrikeFalconSandbox/Playbooks/playbook-Detonate_URL_-_CrowdStrike.yml:377]:
```suggestion
    Should the playbook fail on unsupported file type?
```

**Comment by Itay4** [comment]
@jochman please unskip the integration

File: Packs/CrowdStrikeFalconSandbox/Integrations/integration-CrowdStrikeFalconSandbox.yml [hunk 2]
```diff
@@ -635,12 +622,20 @@ script:
...
         if (version === 'v1') {
             throw 'This command is supported only in API v2.'
         }
-        var response = submitUrl(url, environmentID);
+        try {
+          var response = submitUrl(url, environmentID);
+        } catch (exception) {
+              var notSupported = 'The provided URL resolves to a file.'
+              if (dontThrowErrorOnFileDetonation && exception.contains(notSupported)){
+                  return 'The file format is not supported, use the command "crowdstrike-submit-file-by-url" instead.';
```
Comment (jochman) [inline Packs/CrowdStrikeFalconSandbox/Integrations/integration-CrowdStrikeFalconSandbox.yml:634]: @richardbluestone Please review the return error.

File: Packs/CrowdStrikeFalconSandbox/Integrations/integration-CrowdStrikeFalconSandbox.yml [hunk 3]
```diff
@@ -1334,6 +1329,13 @@ script:
       required: true
       description: Environment ID to submit URL to
       defaultValue: "100"
+    - name: dontThrowErrorOnFileDetonation
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      description: Not throwing error when sending URL that points to a file.
```
Comment (jochman) [inline Packs/CrowdStrikeFalconSandbox/Integrations/integration-CrowdStrikeFalconSandbox.yml:1337]: @richardbluestone Review this description.

File: Packs/CrowdStrikeFalconSandbox/Integrations/integration-CrowdStrikeFalconSandbox_README.md [hunk 4]
```diff
@@ -1870,6 +1870,10 @@
 <td style="width: 472px;">The ID of the environment to submit the URL to.</td>
 <td style="width: 98px;">Required</td>
 </tr>
+<td style="width: 170px;">dontThrowErrorOnFileDetonation</td>
+<td style="width: 472px;">Determine if the command will throw error on unsupported file error. Default is false. .</td>
```
Comment (jochman) [inline Packs/CrowdStrikeFalconSandbox/Integrations/integration-CrowdStrikeFalconSandbox_README.md:1874]: @richardbluestone Review this doc update

File: Packs/CrowdStrikeFalconSandbox/Playbooks/playbook-Detonate_URL_-_CrowdStrike.yml [hunk 5]
```diff
@@ -274,21 +349,34 @@ inputs:
...
     simple: "30"
   required: false
   description: How much time to wait before a timeout occurs  (minutes)
+  playbookInputQuery:
+- key: dontThrowErrorOnFileDetonation
+  value:
+    simple: "false"
+  required: false
+  description: |-
+    Should the playbook fail on unsupported file type?
```
Comment (jochman) [inline Packs/CrowdStrikeFalconSandbox/Playbooks/playbook-Detonate_URL_-_CrowdStrike.yml:377]: @richardbluestone Review this description

File: Packs/CrowdStrikeFalconSandbox/ReleaseNotes/1_0_1.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+##### CrowdStrike Falcon Sandbox
+Added the option to not throw error when sending a url that points to a file in **crowdstrike-submit-url** command.
```
Comment (jochman) [inline Packs/CrowdStrikeFalconSandbox/ReleaseNotes/1_0_1.md:4]: @richardbluestone review rns

**Review by Itay4 — APPROVED** [review]

**Pull request merged**